core/src/main/scala/com/criteo/cuttle/Executor.scala

```diff
@@ -620,6 +629,7 @@ class Executor[S <: Scheduling] private[cuttle] (val platforms: Seq[ExecutionPla
             execution.streams.error(stacktrace.toString)
             atomic {
               implicit tx =>
+                // retain jobs in a recent failures if a last failure happened in [now - retryStrategy.retryWindow, now]
```

Comment:
May I suggest removing both 'a'
// retain jobs in **a** recent failures if **a** last failure happened in [now - retryStrategy.retryWindow, now]

Reply:
You may and I will correct it ty!